libs/openFrameworks/app/ofAppGLFWWindow.cpp

```diff
@@ -1204,7 +1204,7 @@ void ofAppGLFWWindow::mouse_cb(GLFWwindow* windowP_, int button, int state, int
 	if (state == GLFW_PRESS) {
 		action = ofMouseEventArgs::Pressed;
 		instance->buttonPressed=true;
-	} else if (state == GLFW_RELEASE) {
+	} else {
```

Comment:
According to the docs, only `GLFW_RELEASE` / 'GLFW_PRESS` are ever sent.  So until an additional value is added, this avoids possible uninitialized routes.